takwimu/templates/takwimu/_includes/sections/stories/index.html

```diff
@@ -1,7 +1,7 @@
 <!-- Stories Section -->
 
 <section class="stories">
-  <div class="container py-5 border-top border-bottom border-left-0 border-right-0" style="border-style: dashed !important;">
```

Comment:
Emm.. Why have you removed the border? It's on the stories section and not on the updates.

Reply:
@DavidLemayian 
The border in the stories where unnecessary, so it was adding another thicker layer in the updates section which distort the layout plus in the stories section there are no borders

Reply:
@DavidLemayian also both fixes where once the same issue, i divided the task so that i can focus on quick issues first then finish up the task that requires more time

Reply:
@mtuchi The borders are in the design. Do you need glasses? https://zpl.io/2pOJBzj

Reply:
@DavidLemayian i apologize i have not seen this zepline design 
the screenshot that was attached on the issue was different  
![screenshot](https://user-images.githubusercontent.com/4629781/43040702-8f8e3c96-8d42-11e8-9efa-f832f85cb56c.png)

In the future all issue should have the exact template reference in zeplin to avoid this kinds of mistakes 

I will fix this shortly

Reply:
@mtuchi Hmm.. The top one is what needs fixing and the bottom is Zeplin designs we should follow. I can tell because of the icons. But I understand the confusion.